tests/_support/Helper/OpenApiValidatorHelper.php

```diff
@@ -68,11 +68,37 @@ public function haveOpenApiFileWithPathButInvalidHttpMethod(): void
      */
     protected function prepareOpenApiSchema(array $files): void
     {
-        $structure = [
-            'resources' => [
-                'api' => $files,
-            ],
-        ];
-        $this->getSyncApiHelper()->mockDirectoryStructure($structure);
+        $this->getSyncApiHelper()->mockDirectoryStructure(
+            $this->buildStructureByPath($this->getOpenApiSchemaPath(), $files),
+        );
+    }
+
+    /**
+     * @return string
+     */
+    public function getOpenApiSchemaPath(): string
+    {
+        return 'resources/api';
+    }
+
+    /**
+     * @param string $path
+     * @param array $files
+     *
+     * @return array
+     */
+    protected function buildStructureByPath(string $path, array $files): array
```

Comment:
Why this change?

Reply:
In OpenApiValidateConsoleTest tests, we use haveDefaultOpenApiFile, which creates a folder structure according to the structure before specified in prepareOpenApiSchema. To check if the error text matches, we need to know in which folder the tested yml file is located. So, to keep the structure in one place it was decided to store the structure in a string and convert it to a multidimensional array if necessary.